charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -400,4 +400,11 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `backup.podSecurityContextOverride`      | Completely overwrites the contents of the backup pod's security context, ignoring the values provided for `runAsUser`, and `fsGroup`. | Not set |
 | `cronJob.apiVersion`                     | CronJob API version | 'batch/v1' |
 | `awsSecurityGroupPolicies.enabled`      | Enable the creation of SecurityGroupPolicy resources | `false` |
-| `awsSecurityGroupPolicies.policies` | Security Group Policy definitions. `awsSecurityGroupPolicies.enabled` must be `true`  | Not set |
\ No newline at end of file
+| `awsSecurityGroupPolicies.policies` | Security Group Policy definitions. `awsSecurityGroupPolicies.enabled` must be `true`  | Not set |
+
+### Unit Test
+
+| Parameter                      | Description                       | Default         |
+| ------------------------------ | --------------------------------- | --------------- |
+| `unittest.testFramework.image` | Image used to test the framework  | `bats/bats`     |
```

Comment:
```suggestion
| `testFramework.image` | Image used to test the framework  | `bats/bats`     |
```

Maybe just omit the `unittest` prefix to avoid confusion with the helm unit test plugin?

Reply:
The reason I created/named the `unittest` section was for future proofing any additional test values that someone may want to add. Then all the unit test values can be in a single location. I am happy to change and omit the value as to not cause confusion! Makes sense that it would cause confusion to other developers, just wanted to explain my reasoning behind the naming. You know this repository much better so I am happy to take your guidance on the best naming convention! Hope you have a fantastic start to the week and thanks for taking time to review my pull request!